fivetran_provider_async/hooks.py

```diff
@@ -1,13 +1,471 @@
 import asyncio
+import json
 import time
+from time import sleep
 from typing import Any, Dict, cast
 
 import aiohttp
 import pendulum
+import requests
 from aiohttp import ClientResponseError
 from airflow.exceptions import AirflowException
+from airflow.hooks.base import BaseHook
 from asgiref.sync import sync_to_async
-from fivetran_provider.hooks.fivetran import FivetranHook
+from requests import exceptions as requests_exceptions
+
+
+class FivetranHook(BaseHook):
```

Comment:
Why not inherit from `from fivetran_provider.hooks.fivetran import FivetranHook` and use that class instead for `FivetranHookAsync` because most of the methods in sync once are the same as of `from fivetran_provider.hooks.fivetran import FivetranHook`?

Reply:
as mentioned in the description fivetran_provider has been archived and I have just moved code from that repo to here